sst/sstable.go

```diff
@@ -20,9 +20,34 @@ type SSTable struct {
 	numDeletes   uint32
 	indexOffset  uint32
 	creationTime uint64
-	data         []byte
+
+	//  data contains
+	//  ╭──────┬──────────────╮
+	//  │format│metadataOffset│
+	//  ├──────┼──────────────┤
+	//  │1 byte│ 4 bytes      │
+	//  ╰──────┴──────────────╯
+	//  ╭─────────┬────────────────┬────────────┬──────────────────╮
+	//  │keyLength│ key            │ valueLength│ value            │
+	//  ├─────────┼────────────────┼────────────┼──────────────────┤ ... repeat KV pairs
+	//  │4 bytes  │ keyLength bytes│ 4 bytes    │ valueLength bytes│
+	//  ╰─────────┴────────────────┴────────────┴──────────────────╯
+	//  ╭───────────────────────────────────────────┬──────────╮
```

Comment:
It might be worth saying this is the index.

Reply:
Updated the comment mentioning the index